include/seqan3/alphabet/aminoacid/all.hpp

```diff
@@ -31,30 +31,57 @@
 // DAMAGE.
 //
 // ============================================================================
-// Author: Sara Hetzel <sara.hetzel AT fu-berlin.de>
-// ============================================================================
-
-#include <gtest/gtest.h>
-#include <seqan3/alphabet/aminoacid/aa27.hpp>
-#include <seqan3/alphabet/aminoacid/aa27_container.hpp>
 
-using namespace seqan3;
+/*!\file alphabet/aminoacid/all.hpp
+ * \ingroup aminoacid
+ * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
+ * \brief Meta-header for the aminoacid submodule; includes all headers from alphabet/aminoacid/.
+ */
 
-TEST(alphabet_aminoacid_aa27_container_test, vector)
-{
-    aa27_vector v{{aa27::A}, {aa27::C}, {aa27::G}, {aa27::T}};
-
-    EXPECT_EQ(v[0], aa27::A);
-    EXPECT_EQ(v[1], aa27::C);
-    EXPECT_EQ(v[2], aa27::G);
-    EXPECT_EQ(v[3], aa27::T);
-}
+#include <seqan3/alphabet/aminoacid/aa27.hpp>
 
-TEST(alphabet_aminoacid_aa27_container_test, string)
-{
-    aa27_string s{{aa27::A}, {aa27::C}, {aa27::G}, {aa27::T}};
-    EXPECT_EQ(s[0], aa27::A);
-    EXPECT_EQ(s[1], aa27::C);
-    EXPECT_EQ(s[2], aa27::G);
-    EXPECT_EQ(s[3], aa27::T);
-}
+/*!\defgroup aminoacid
+ * \brief Contains the amino acid alphabets and functionality for translation from nucleotide.
+ * \ingroup alphabet
+ *
+ * \par Introduction
+ * Amino acid sequences are an important part of bioinformatic data processing and used by many applications
+ * and while it is possible to represent them in a regular std::string, it makes sense to have specialised data
+ * structures in most cases. This sub-module offers the 27 letter aminoacid alphabet as well as a reduced version
```

Comment:
Maybe one more sentence that that explains, why 27? I.e. 20 canonical/iupac plus 2 extra proteinogenic, four wildcard and terminator...?